indexing-service/src/main/java/org/apache/druid/indexing/common/task/KillUnusedSegmentsTask.java

```diff
@@ -225,7 +225,14 @@ public TaskStatus runTask(TaskToolbox toolbox) throws Exception
       // Fetch the load specs of all segments overlapping with the given interval
       final Set<Map<String, Object>> usedSegmentLoadSpecs = toolbox
           .getTaskActionClient()
-          .submit(new RetrieveUsedSegmentsAction(getDataSource(), getInterval(), null, Segments.INCLUDING_OVERSHADOWED))
+          .submit(new RetrieveUsedSegmentsAction(
```

Comment:
Format: would be easier to read if the unused intervals were pre-constructed and the constructor was moved to a new line.
```suggestion
          .submit(
             new RetrieveUsedSegmentsAction(getDataSource(), null, unusedIntervals, Segments.INCLUDING_OVERSHADOWED)
          )
```

Reply:
Done